astroid/brain/brain_namedtuple_enum.py

```diff
@@ -379,7 +379,11 @@ def infer_enum_class(node: nodes.ClassDef) -> nodes.ClassDef:
                 continue
 
             inferred_return_value = None
-            if isinstance(stmt, (nodes.AnnAssign, nodes.Assign)):
+            if (
+                isinstance(stmt, nodes.Assign)
+                or isinstance(stmt, nodes.AnnAssign)
+                and stmt.value
+            ):
                 if isinstance(stmt.value, nodes.Const):
```

Comment:
Nitpick but I would do this (it feels simpler to me):
```suggestion
            if isinstance(stmt, (nodes.AnnAssign, nodes.Assign)):
                if stmt.value and isinstance(stmt.value, nodes.Const):
```

Reply:
👍